notebooks/explainable-ai-1-basic/README.md

```diff
@@ -0,0 +1,21 @@
+# eXplainable AI (XAI) for OpenVINO™ IR Models
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/explainable-ai-basic/explainable-ai-basic.ipynb)
```

Comment:
path for colab link is not matched with notebook path relative to repository root
```suggestion
[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/explainable-ai-1-basic/explainable-ai-1-basic.ipynb)
```

Reply:
good catch, thanks!